docs/index.md

```diff
@@ -114,7 +114,14 @@ First install yotta's dependencies using your system's package manager, for
 example on Debian and Ubuntu:
 
 ```sh
+# install development tool dependencies
 sudo apt-get install python-pip cmake build-essential ninja-build python-dev
+
+# update pip to latest release
+sudo apt-get remove python-pip
+curl -o get-pip.py https://bootstrap.pypa.io/get-pip.py
+sudo python get-pip.py
+hash -r
```

Comment:
Isn't executing some arbitrary downloaded python script as root somewhat of an anti-pattern?

Should we not be adding some other repository to provide the latest pip, or using easy_install ?